demos/index.php

```diff
@@ -58,10 +58,23 @@
 // list all filestore files
 $c2 = $columnsLayout->addColumn();
 Header::addTo($c2, ['All Filestore Files']);
-$gr = Grid::addTo($c2, [
+$gr = Crud::addTo($c2, [
     'paginator' => false,
 ]);
 $files = new File($app->db, ['flysystem' => $filesystem]);
+$files->removeUserAction('add');
+$files->removeUserAction('edit');
+$files->removeUserAction('delete');
+$files->addUserAction('cleanup_drafts', [
+    'callback' => function ($m) {
+        $m->cleanupDrafts();
+
+        return 'Draft files are deleted.';
+        // return $gr->jsReload(); // @todo this way it's impossible?
```

Comment:
return JsBlock?

Reply:
That doesn't work. Then it just shows notification with text "Done!" (Crud::defaultMsg).
That's how it is implemented in Atk4\Ui\Crud.

Reply:
grid callback will solve this